src/ui/component/splash/view.jsx

```diff
@@ -148,10 +148,10 @@ export default class SplashScreen extends React.PureComponent<Props, State> {
         });
       }
     } else if (wallet && wallet.blocks_behind > 0) {
-      const format = wallet.blocks_behind === 1 ? '%s block behind' : '%s blocks behind';
+      const amountBehind = wallet.blocks_behind === 1 ? '%amountBehind% block behind' : '%amountBehind% blocks behind';
       this.setState({
         message: __('Blockchain Sync'),
-        details: `${__('Catching up...')} (${__(format, wallet.blocks_behind)})`,
+        details: `${__('Catching up...')} (${__(amountBehind, { amountBehind: wallet.blocks_behind })})`,
```

Comment:
This should just be `__(` now, not `${__(`, right?

(Also, this is where `__n` could be useful if we want it)

Reply:
We still need the `$` because it's wrapped in a template string